README.md

```diff
@@ -1,31 +1,70 @@
+
+
+Quickstart
 -----------
 
-For a thirty second quickstart, look at http://www.rethinkdb.com/docs/guides/quickstart
+For a thirty-second RethinkDB quickstart, check out  [rethinkdb.com/docs/quickstart](http://www.rethinkdb.com/docs/quickstart).
+
+Or get started right away, with one of our language specific guides:
+
+* [Ten-minute guide with RethinkDB and Python](http://rethinkdb.com/docs/guide/python/)
+* [Ten-minute guide with RethinkDB and Ruby](http://rethinkdb.com/docs/guide/ruby/)
+* [Ten-minute guide with RethinkDB and JavaScript](http://rethinkdb.com/docs/guide/javascript/)
 
-Get help
+Besides our three official drivers, we also have many [community drivers](http://rethinkdb.com/docs/install-drivers/) supported by the RethinkDB community. Here's a few:
+
+* [C#/.NET rethinkdb-net](https://github.com/mfenniak/rethinkdb-net) - [@mfenniak](https://github.com/mfenniak)
+* [Clojure clj-rethinkdb](https://github.com/apa512/clj-rethinkdb) - [@apa512](https://github.com/apa512)
+* [Go      GoRethink](https://github.com/dancannon/gorethink) - [@dancannon](https://github.com/dancannon)
+* [Haskell haskeel-rethinkdb](https://github.com/atnnn/haskell-rethinkdb) - [@atnnn](https://github.com/atnnn)
+* [PHP     php-rql](https://github.com/danielmewes/php-rql) - [@danielmewes](https://github.com/danielmewes)
+* [Scala   rethink-scala](https://github.com/kclay/rethink-scala) - [@kclay](https://github.com/kclay)
+
+
+Need help?
 --------
 
-Check out www.rethinkdb.com/community
+If you need help, a great place to start is  [rethinkdb.com/community](www.rethinkdb.com/community). Here you can find out how to ask us questions, reach out to us, or [report an issue](https://github.com/rethinkdb/rethinkdb/issues). As well, find all the places we frequent online and which conference or meetup you might be able to meet us next.
+
+If you need help right now, you can also find us [on Twitter](http://twitter.com/rethinkdb), via IRC at [#rethinkdb](irc://chat.freenode.net/#rethinkdb) on Freenode, or at [gitter.im/rethinkdb/rethinkdb](https://gitter.im/rethinkdb/rethinkdb).
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/rethinkdb/rethinkdb?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+Do you use RethinkDB? Send us a story!
+-----------------
+ We love hearing all the awesome ways people are using RethinkDB and we want to hear from you too. Send us your story and we'll send a package your way with  RethinkDB swag.
+
+[rethinkdb.com/community/shirts-for-stories](http://rethinkdb.com/community/shirts-for-stories)
 
 Contributing
 -----------------
 
-Check out our [contributing guidelines](CONTRIBUTING.md).
+RethinkDB was built by a dedicated team but would not have been possible without the support and contributions of hundreds of people from all of the world. We could use your help too! Check out our [contributing guidelines](CONTRIBUTING.md) to get started.
```

Comment:
Missing comma:

RethinkDB was built by a dedicated team, but would not have been possible without the support and contributions of hundreds of people from all of the world. We could use your help too!